snapcraft/storeapi/__init__.py

```diff
@@ -91,6 +92,10 @@ def __init__(self, conf, root_url):
         self.conf = conf
         self.root_url = root_url
         self.session = requests.Session()
+        # Setup max retries for all store URLs and the CDN
+        self.session.mount('http://', HTTPAdapter(max_retries=5))
+        self.session.mount('https://', HTTPAdapter(max_retries=5))
```

Comment:
This is simple enough, we might as well see if it fixes the issue. If it doesn't we can make it wait a little longer.